package.json

```diff
@@ -56,13 +56,13 @@
     "react-dnd-test-backend": "^2.6.0",
     "react-docgen": "^2.21.0",
     "react-dom": "^16.8.6",
-    "react-router-dom": "^5.0.0",
+    "react-router-dom": "^5.0.1",
     "recharts": "^1.6.2",
     "style-loader": "^0.23.1",
     "terser-webpack-plugin": "^1.3.0",
     "ts-jest": "^24.0.1",
-    "typescript": "~3.5.1",
-    "webpack": "^4.32.2",
-    "webpack-dev-server": "^3.4.1"
+    "typescript": "^3.5.2",
```

Comment:
Should keep the `~` range.

Reply:
I think we can keep `^` for now. I will add `~` back if we run into any issues.